crates/katana/Cargo.toml

```diff
@@ -15,6 +15,7 @@ katana-core = { path = "core" }
 katana-rpc = { path = "rpc" }
 starknet_api.workspace = true
 yansi.workspace = true
+clap_complete = "4.3.2"
```

Comment:
lets have this as workspace dependency

Reply:
oh i just used `cargo add` didnt see we use workspace dependencies xP. fixed